### DISCUSSION TITLE
How do I get a file within a file?

### DISCUSSION THREAD

**Discussion by Nurbol-Xan** [community/community#27593] [Other Feature Feedback, Questions, & Ideas]
**For example**: a **folder** given to git contains a **folder**,
How can I get the **folder** inside without doing a git clone of the whole **file**?

**Comment by ihnatenkoo** [comment]
Hello, use this service which can donwload certain folder from git repository. 
Just copy the link to this folder and paste to the service.
_**https://downgit.github.io/#/home**_

**Reply by Nurbol-Xan** [reply]
Thanks for the reply!

**Comment by michaelimasih** [comment]
Mi account ID 6584892478
